api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -211,6 +211,13 @@ message JwtProvider {
   // Specify the clock skew in seconds when verifying JWT time constraint,
   // such as `exp`, and `nbf`. If not specified, default is 60 seconds.
   uint32 clock_skew_seconds = 10;
+
+  // This field only applies to :ref:`forward_payload_header <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtProvider.forward_payload_header>`.
```

Comment:
Please move this field next to `forward_payload_header` field (right after it).

Reply:
Done.